node/sync.go

```diff
@@ -176,32 +179,35 @@ func (p *syncProviderWrapper) updatePodStatus(ctx context.Context, podFromKubern
 		return nil
 	}
 
+	if podFromKubernetes.Status.Phase != corev1.PodRunning && time.Since(podFromKubernetes.ObjectMeta.CreationTimestamp.Time) <= time.Minute {
+		span.SetStatus(err)
```

Comment:
Shouldn't this be `span.SetStatus(statusErr)`, because `err` gets rewritten by `cache.MetaNamespaceKeyFunc`?

Reply:
:( Yes. Thanks